Tasks/sum.js

```diff
@@ -1,20 +1,14 @@
-// Sum all numbers from an array
-
-const k = (sum = function (s = () => {}) {
-  sum = [0];
-  let k = 5;
-  for (i of s) {
-    let t = typeof i;
-    if (t === 'number') {
-      if (sum.length > 0) {
-        const new_Sum = sum[sum.length - 1] + i;
-        sum.push(new_Sum);
+//Step 4
+"use strict";
+const sum = (argArray) => {
+  const res = [0];
+  argArray.map((arrayElem) => {
```

Comment:
We have `map` to convert array to new one. Do not use map to change outer array.